frontend/src/container/TriggeredAlerts/index.tsx

```diff
@@ -1,91 +1,47 @@
 import getTriggeredApi from 'api/alerts/getTriggered';
 import Spinner from 'components/Spinner';
-import { State } from 'hooks/useFetch';
 import { useNotifications } from 'hooks/useNotifications';
-import { useCallback, useEffect, useState } from 'react';
+import { useEffect } from 'react';
 import { useTranslation } from 'react-i18next';
-import { PayloadProps } from 'types/api/alerts/getTriggered';
+import { useQuery } from 'react-query';
+import { useSelector } from 'react-redux';
+import { AppState } from 'store/reducers';
+import AppReducer from 'types/reducer/app';
 
 import TriggerComponent from './TriggeredAlert';
 
 function TriggeredAlerts(): JSX.Element {
-	const [groupState, setGroupState] = useState<State<PayloadProps>>({
-		error: false,
-		errorMessage: '',
-		loading: true,
-		success: false,
-		payload: [],
-	});
 	const { t } = useTranslation(['common']);
 	const { notifications } = useNotifications();
+	const { user } = useSelector<AppState, AppReducer>((state) => state.app);
```

Comment:
Since only `userId` is being used from the store, we can extract just that
```suggestion
	const userId = useSelector<AppState, AppReducer>((state) => state.app.user?.userId);
```